apollo/build.gradle.kts

```diff
@@ -590,7 +604,7 @@ kotlin {
                 implementation("fr.acinq.secp256k1:secp256k1-kmp-jni-jvm:0.11.0")
                 implementation("fr.acinq.secp256k1:secp256k1-kmp-jni-android:0.14.0")
                 implementation("com.google.guava:guava:30.1-jre")
-                implementation("org.bouncycastle:bcprov-jdk15on:1.68")
+                implementation("org.bouncycastle:bcprov-jdk18on:1.80")
```

Comment:
You still have `implementation("org.bouncycastle:bcprov-jdk15on:1.68")` in line 634